app/views/users/show.html.erb

```diff
@@ -1,2 +1,2 @@
-<h1><%= t(".welcome", name: user.name) %></h1>
-<h2><%= t(".profile_information", email: user.email) %></h2>
+<div><%= t(".welcome") %></div>
```

Comment:
I would prefer to use a heading tag here.

Reply:
Done ins sha: 1ea0d5e